cargo-transient.el

```diff
@@ -291,21 +280,21 @@ GENERATE-COLLECTION is a function which returns a list of strings."
 (transient-define-prefix cargo-transient--prefix-run ()
   "Run `cargo run`."
   :man-page "cargo-run"
-  [cargo-transient--group-target-selection
+  ["Target Selection"
    (cargo-transient--arg-bin)
    (cargo-transient--arg-example)]
-  [cargo-transient--group-feature-selection
+  ["Feature Selection"
    (cargo-transient--arg-features)
    (cargo-transient--arg-all-features)
    (cargo-transient--arg-no-default-features)]
-  [cargo-transient--group-compilation-options
+  ["Compilation Options"
    (cargo-transient--arg-release)]
-  [7 cargo-transient--group-manifest-options
+  ["Manifest Options"
    (cargo-transient--arg-offline)]
   [cargo-transient--group-arguments
```

Comment:
^ leftover reference to the now-deleted const

Reply:
Thanks, fixed!